application/src/main/java/org/opentripplanner/street/model/edge/TemporaryPartialStreetEdge.java

```diff
@@ -83,6 +83,6 @@ public int getInAngle() {
    */
   @Override
   public int getOutAngle() {
-    return parentEdge.getInAngle();
+    return parentEdge.getOutAngle();
```

Comment:
This is interesting. I wonder if this has caused any problems? Maybe it isn't used at all? Good catch

Reply:
Can you remove the excess whitespace in the Javadoc for getInAngle and getOutAngle

Reply:
This code is heavily used but it has never been tested. Perhaps I should add a test here.

Reply:
One thing that came to mind is that you could separate the bug fix into its own PR, this way it would be easier to see whether the change to the internal angle representation or the bug fix causes the speed ups.

Reply:
If you decide to do this, you can add me as a reviewer on the bug fix as well